packages/manager/modules/pci/src/components/project/users/translations/Messages_fr_FR.json

```diff
@@ -1,5 +1,5 @@
 {
-  "pci_projects_project_users_title": "Utilisateurs",
+  "pci_projects_project_users_title": "Users & Roles",
```

Comment:
**question:** If content is not meant to be translated, couldn't we use it directly as a constant?